app/gui/src/controller/searcher.rs

```diff
@@ -1442,12 +1447,21 @@ impl Searcher {
 
 // === Searcher helpers ===
 
+enum PrivateEntriesVisibility {
+    Show,
+    Hide,
+}
+
```

Comment:
We keep the visibility as an enum only here, and shortly after converting from bool is converted to bool again. I would keep either bools or enums everywhere. I would stay with bools to keep the code a bit simpler.